nrf_security/sst_settings/Kconfig

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+config SST_SETTINGS
+	bool "Settings module for Secure Storage"
```

Comment:
I think we are missing Settings dependency here. 
Do not allow enabling this option if settings subsystem is not enabled.